Classes/Domain/Model/Feedback/Operations/UpdateNodeInfo.php

```diff
@@ -33,6 +33,28 @@ class UpdateNodeInfo extends AbstractFeedback
 
     protected $isRecursive = false;
 
+    protected $baseNodeType = null;
+
+    /**
+     * Set the baseNodeType
+     *
+     * @param string $baseNodeType
+     */
+    public function setBaseNodeType($baseNodeType)
```

Comment:
```suggestion
    public function setBaseNodeType(string $baseNodeType): void
```

Reply:
Absolutely ;)

Reply:
Did you just comment on your own suggestion? :)